src/models/changeLog/asyncAction.ts

```diff
@@ -0,0 +1,15 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { fetchMaterialsChange } from 'old/lib/utilities/API/api';
+import { IChangeLog } from './types';
+
+export const getChangeLog = createAsyncThunk(
```

Comment:
but where this action is used?

Reply:
this acition im used in reducet like extraReducer 
```suggestion
const changeLog = createSlice({
  name: 'changesLog',
  initialState,
  reducers: {
    setChangesSize: (state, action: PayloadAction<number>) => {
      const setChangeObj = state;
      setChangeObj.size = action.payload;
    },
  },
  extraReducers: {
    getAsyncActionsReducer(getChangeLog as any),
  },
});
```